source/features/file-finder-buffer.tsx

```diff
@@ -60,11 +60,6 @@ function pjaxCompleteHandler(): void {
 function init(): Deinit {
 	window.addEventListener('pjax:start', pjaxStartHandler);
 	window.addEventListener('pjax:complete', pjaxCompleteHandler);
-
-	return () => {
-		window.removeEventListener('pjax:start', pjaxStartHandler);
-		window.removeEventListener('pjax:complete', pjaxCompleteHandler);
-	};
```

Comment:
:man_facepalming:   :man_facepalming:   :man_facepalming: 

I haven't found any other feature that have been similarly broken by #5422, so I think we're good.